spec/spec_helper.rb

```diff
@@ -1 +1,9 @@
 require 'puppetlabs_spec_helper/module_spec_helper'
+require 'rspec-puppet'
```

Comment:
I just tested it. You are right that the "require rspec-puppet" is obsolete (it is included in the puppetlabs spec helper). 
Nevertheless the rake task does not succeed if the directories for the fixtures are not defined (at least in my local environment).